### PR TITLE
Pictogram: Multiple SVG folder processing, add decorative pictogram, etc

### PR DESCRIPTION
**Pull request by jonnystening**
- rename pictogram `type` prop as `mode`
- add sizing props (`width` and `height` as per `@westpac/Symbols`)
- update buildExports helper to allow processing of multiple export folders (eg. 'decorative/' and 'informative/')
- add decorative pictogram, process along with informative pictograms
- update examples

The change to `buildExports.js` will allow you to…

**import when there’s only one folder to process (eg. icons)…**
`import { AddIcon } from '@westpac/icon';` or with code-splitting `import { AddIcon } from '@westpac/icon/AddIcon';` (as before)

**and import when there’s multiple folders to process (note subfolder)…**
`import { WalletPictogram } from '@westpac/pictogram/informative';`  or with code-splitting `import { WalletPictogram } from '@westpac/pictogram/informative/WalletPictogram';`

**and…**
`import { WBCBankCardPictogram } from '@westpac/pictogram/decorative';` or with code-splitting `import { WBCBankCardPictogram } from '@westpac/pictogram/decorative/WBCBankCardPictogram';`

### CODE REVIEW

File: components/pictogram/examples/00-default.js [hunk 1]
```diff
@@ -4,24 +4,39 @@ import { GEL, jsx } from '@westpac/core';
 import * as components from '@westpac/pictogram';
 import { Grid, Cell, Name } from './_util';
 
-const pictograms = Object.keys(components);
+const informative = Object.keys(components).filter((s) => !s.startsWith('WBC'));
+const decorative = Object.keys(components).filter((s) => s.startsWith('WBC'));
 
 function Example({ brand }) {
```
[the remark this answers is not in the document]
Comment (jonnystening) [reply components/pictogram/examples/00-default.js:10]: Oh yeah, i left that to come back to (hard coded for WBC at the moment). 

These components aren't multi-brand. I just need a nicer way to filter out the components that come from `decorative/` (the component names will always begin with brand codes, but I'd love an more elegant way)

File: components/pictogram/src/_utils.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export const colorMap = (COLORS) => ({
+export const colourMap = (COLORS) => ({
```
[the remark this answers is not in the document]
Comment (jonnystening) [reply components/pictogram/src/_utils.js:1]: Ok ok

File: components/pictogram/examples/00-default.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dominikwilkowski) [reply components/pictogram/examples/00-default.js:10]: yeah but fixing it now is the right thing and only a couple lines. Let's fix it now

**Review by jaortiz — APPROVED** [review]
LGTM

**Pull request merged**